### PR TITLE
[8.0][ADD] Prevent conflicting companies on journal and analytic journal.

### PR DESCRIPTION
**Pull request by NL66278**
Prevent problems with analytic journals (and later entries) in a different company, than the main journal.

### CODE REVIEW

File: account_analytic_restrict_journal_company/README.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :alt: License
+
+Restrict Analytic Journal on Company
```
Comment (elicoidal) [inline account_analytic_restrict_journal_company/README.rst]: Add a line before the title as well

File: account_analytic_restrict_journal_company/README.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+    aj.id, aj.name, aj.company_id,
+    aaj.id, aaj.name, aaj.company_id, rc.name
+from account_journal aj 
+join account_analytic_journal aaj on aj.analytic_journal_id = aaj.id
+join res_company rc on aj.company_id = rc.id
+where aj.company_id != aaj.company_id;
+```
+
+
+More info
```
Comment (elicoidal) [inline account_analytic_restrict_journal_company/README.rst]: Remove this section.

Comment (elicoidal) [reply account_analytic_restrict_journal_company/README.rst]: And add runbot information following newest template

File: account_analytic_restrict_journal_company/__openerp__.py [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# -*- coding: utf-8 -*-
+# © 2017 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+{
+    'name': 'Account Analytic Restrict Journal Company',
+    'version': '8.0.0.1.0',
```
Comment (elicoidal) [inline account_analytic_restrict_journal_company/__openerp__.py]: 8.0.1.0.0

**Review by elicoidal — CHANGES_REQUESTED** [review]
some details

**Comment by NL66278** [comment]
@elicoidal Hi, changes requested have been done!

File: account_analytic_restrict_journal_company/README.rst [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+
+Both the chart of accounts and analytic accounts should be kept strictly
+separate for different companies.
+
+Installation
+============
+
+After installing this module no more journals can be created that refer to
+the analytic journals of another company. However existing wrong
+configurations are not (and cannot) be automatically corrected. To check
```
Comment (elicoidal) [inline account_analytic_restrict_journal_company/README.rst]: "are not (and cannot be)"

**Review by elicoidal — APPROVED** [review]

**Review by andhit-r — APPROVED** [review]

**Comment by lfreeke** [comment]
Can we merge this now?

**Pull request closed without merging**